Makefile

```diff
@@ -7,11 +7,12 @@ timed:
 	time raco make main.rkt
 
 test: all
+	# cd src/backend-c/runtime/tests && make all && ./tests
```

Comment:
I would make this line call the `make -c src/backend-c/runtime test` and have that makefile setup anything that needs to be done. That would be more extensible for other runtime tests and not clutter this space.

Reply:
Done.